src/Tabs/Tabs.js

```diff
@@ -0,0 +1,243 @@
+    }
+
+    scrollToTab(tab, scrollProps = { callback: this.animatedScrollCallback }) {
+        this.removeSideScrollListener()
+
+        animatedSideScroll({
+            targetEl: tab,
+            scrollBox: this.scrollBox.current,
+            ...scrollProps,
+        })
+    }
+
+    toggleScrollButtonVisibility = () => {
+        const { scrollLeft, offsetWidth } = this.scrollBox.current
+        const { offsetWidth: areaOffsetWidth } = this.scrollArea.current
+        const scrolledToStart = scrollLeft === 0
+        const scrolledToEnd = scrollLeft + offsetWidth === areaOffsetWidth
+
+        if (
+            this.state.scrolledToStart !== scrolledToStart ||
+            this.state.scrolledToEnd !== scrolledToEnd
+        ) {
+            this.setState({
+                scrolledToStart,
+                scrolledToEnd,
+            })
+        }
+    }
+
+    render() {
+        const {
+            isScrollingRequired,
+            scrolledToStart,
+            scrolledToEnd,
+            showTabIndicator,
+        } = this.state
+        const { className, fixed, children, selected } = this.props
+
+        return (
+            <div className={className} ref={this.container}>
+                <TabBar
+                    isScrollingRequired={isScrollingRequired}
+                    fixed={fixed}
+                    scrollLeft={this.scrollLeft}
+                    scrollRight={this.scrollRight}
+                    scrolledToStart={scrolledToStart}
+                    scrolledToEnd={scrolledToEnd}
+                    scrollBoxRef={this.scrollBox}
+                    scrollAreaRef={this.scrollArea}
+                    marginBottom={-this.horizontalScrollbarHeight}
+                >
+                    {Children.map(children, (child, index) =>
+                        cloneElement(child, {
+                            selected: index === selected,
+                            ref: this.tabRefs[index],
+                            fixed,
+                        })
+                    )}
+                    <TabIndicator
+                        getSelectedTabRef={this.getSelectedTabRef}
+                        visible={showTabIndicator}
+                    />
+                </TabBar>
+                <style jsx>{`
+                    div {
+                        width: 100%;
+                        display: flex;
+                        box-sizing: border-box;
+                        flex-shrink: 0;
+                        flex-direction: row;
+                        background-color: ${colors.white};
+                        box-shadow: inset 0 -1px 0 0 ${colors.grey400};
+                    }
+                `}</style>
+            </div>
+        )
+    }
+}
+
+Tabs.propTypes = {
+    className: propTypes.string,
+    selected: propTypes.number.isRequired,
+    fixed: TabBar.propTypes.fixed,
+    children: propTypes.arrayOf(instanceOfComponent(Tab)),
+}
+
+Tabs.defaultProps = {
+    fixed: false,
```

Comment:
When using `TabBar.propTypes.fixed` I guess it makes sense to also use `TabBar.defaultProps.fixed` here